### ISSUE TITLE
Question about time

### ISSUE THREAD

**Issue by lyt719** [Cli212/flax-BERT#1]
May I ask you, after using the flax and jax framework, the training of BERT has improved in efficiency?

**Comment by Cli212** [comment]
Yes. According to my test, the latency is much faster for jax (flax) compared to Pytorch and Tensorflow. And the memory cost of jax is a little bit larger than those two. However, this is just the result I got two years ago, don't know if things change now.

**Comment by lyt719** [comment]
> 

Thanks!
